NearbyShare/NearbyConnection.swift

```diff
@@ -119,6 +120,7 @@ class NearbyConnection{
 				self.protocolError()
 				return
 			}
+            SleepManager.shared.disableSleep()
```

Comment:
This callback is called for every incoming packet (this method is the first step in receiving a packet, it reads the int32 length prefix). It would work better to call this once per file transfer, after the user has accepted it.

Reply:
Ah oh, yes I do it. I hav overseen this comment.

Reply:
Yes I moved the function call to another place